innopoints/views/notification.py

```diff
@@ -0,0 +1,35 @@
+"""Views related to notifications.
+
+- GET   /notifications
+- PATCH /notifications/{notification_id}/read
+"""
+
+from flask import abort
+from flask_login import login_required, current_user
+
+from innopoints.blueprints import api
+from innopoints.extensions import db
+from innopoints.models import Notification
+from innopoints.schemas import NotificationSchema
+
+
+@api.route('/notifications')
+@login_required
+def get_notifications():
+    """Gets all notifications of the current user."""
+    query = Notification.query.filter_by(recipient_email=current_user.email)
+    notifications = query.all()
+    schema = NotificationSchema(many=True)
+    return schema.jsonify(notifications)
```

Comment:
Not sure if the `notifications` variable is worth it

Reply:
right. it was for debugging purposes and didn't bother improving